Realm/RLMObjectSchema.mm

```diff
@@ -188,6 +199,24 @@ + (NSArray *)propertiesForClass:(Class)objectClass isSwift:(bool)isSwiftClass {
         }
     }
 
+    if (NSArray *optionalProperties = [objectUtil getOptionalPropertyNames:swiftObjectInstance]) {
```

Comment:
I'd prefer to get rid of the above conditional, and I don't mind the stub implementations, but I'm willing to change it -- we already have some stub implementations in the objc util class